src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,17 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+    public Bulldozer(String name) {
+        super(name);
+    }
```

Comment:
No need to add the field 'name'. Just use the name as string in methods

Reply:
this was done for scalability